pkg/types/types.go

```diff
@@ -11,10 +11,19 @@ type Severity int
 
 type VendorSeverity map[string]Severity
 
+type CVSS struct {
+	V2      string  `json:"v2,omitempty"`
```

Comment:
I suggested V2Vector because V2 doesn’t mean vector, but do you prefer V2?

Reply:
I have no preference, I updated as you requested: https://github.com/aquasecurity/trivy-db/pull/48/commits/14729673df77fc74e1424c295102f05dc629f996